treeherder/webapp/api/runnable_jobs.py

```diff
@@ -54,6 +74,39 @@ def list(self, request, project):
                 'state': 'runnable',
                 'result': 'runnable'})
 
+        for label, node in tc_graph.iteritems():
+            build_platform = node['task']['extra']['treeherder']['build']['platform']
+            job_type_name = node['task']['metadata']['name']
```

Comment:
Some of these attributes are referred to a lot, it may be worth doing something like this to avoid the repetition:
```python
task_metadata = node['task']['metadata']
treeherder_options = node['task']['extra']['treeherder']
```
...and then using them below.